pkgs/applications/video/vlc/default.nix

```diff
@@ -8,9 +8,11 @@
 , libmtp, unzip, taglib, libkate, libtiger, libv4l, samba, libssh2, liboggz
 , libass, libva, libdvbpsi, libdc1394, libraw1394, libopus
 , libvdpau, libsamplerate, live555, fluidsynth, wayland, wayland-protocols
+, libXpm, freetype, libXext, libXinerama
```

Comment:
I thing it would be better to use `xorg` actually just to keep the consistency. I will give an example below.